keras_cv/models/__internal__/darknet_utils.py

```diff
@@ -24,24 +24,24 @@
 from tensorflow.keras import layers
 
 
-def DarknetConvBlock(
+def apply_darknet_conv_block(
```

Comment:
This is still returning a sequential, not applying a block to an input Tensor.

Looking at how these blocks are handled, I actually think we should just keep these all as `Layers` for now and leave `darknet_utils` unchanged.

My apologies for the churn, but seeing this change here I think this will get pretty awkward for the darknet blocks, and it will also break our pre-trained weights for DarkNet and CSPDarkNet